web/src/components/application-detail-page/application-detail/index.tsx

```diff
@@ -302,7 +302,11 @@ export const ApplicationDetail: FC<ApplicationDetailProps> = memo(
 
               {app.syncState && (
                 <SyncStateReason
-                  summary={app.syncState.shortReason}
+                  summary={
+                    app.syncState.shortReason
+                      ? app.syncState.shortReason
```

Comment:
does this mean there is a case in which shortReason exists? In such case, please ensure that the shortReason exists in all cases, we want that reason to be shown at this place.